docs/reference/mapping.asciidoc

```diff
@@ -96,7 +96,14 @@ causing a mapping explosion:
     all nested fields, defaults to 10000. Indexing one document with an array of
     100 objects within a nested field, will actually create 101 documents, as
     each nested object will be indexed as a separate hidden document.
-
+`index.mapping.field.name.length.limit`::
```

Comment:
I think this should be named: `index.mapping.field_name_length.limit`. We don't need a namespace for every component.